### PR TITLE
Add Spaceship

### LINKED ISSUE #9637: Icon Request: Spaceship (spaceship.com)
### Brand Name

Spaceship

### Website

https://www.spaceship.com/

### Popularity Metric

[Similarweb Global Rank: 128,231](https://www.similarweb.com/website/spaceship.com/#overview)

### Official Resources for Icon and Color

SVG: spaceship
Color: `#394EFF`

### Additional Comments

[Spaceship.com](https://spaceship.com/) is a subsidiary of [Namecheap.com](https://namecheap.com/), a very popular domain registrar.

### Contributing

No

### PR DESCRIPTION
**Pull request by zcraber**
spaceship

**Issue:** closes #9637

**Similarweb rank:** [128,231](https://www.similarweb.com/website/spaceship.com/#overview)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Logo taken from the header. Brand colour `#394EFF`.

### CODE REVIEW

File: icons/spaceship.svg
Comment (PratyushJayachandran) [inline icons/spaceship.svg:1]: The contributed icon when compared to the source shows [slight difference as seen here!](https://github.com/simple-icons/simple-icons/assets/35000807/a37c8cc3-8236-443c-97eb-6419848d9c5e)
Can you please give it another go @zcraber ?

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

File: icons/spaceship.svg
Comment (zcraber) [reply icons/spaceship.svg:1]: I couldn't see this difference in my testing. I'm using [this](https://user-images.githubusercontent.com/61133303/276521562-945432e1-72e2-4bdf-ac17-d15e65124062.png) source logo taken from the website header using [SVG Gobbler](https://github.com/rossmoody/svg-gobbler) extension.

image

Comment (PratyushJayachandran) [reply icons/spaceship.svg:1]: I will ask someone else to take a look at this one @simple-icons/icon-review

**Review by adamrusted — CHANGES_REQUESTED** [review]
Icon looks to be a perfect match, thanks @zcraber!
Unfortunately, there are an excess of points either side of the icon - some of which were present in the original - but we can probably clean those up before merging!
image

**Comment by adamrusted** [comment]
No stress @zcraber - perhaps one of our reviewers (discounting myself, I can NEVER get these kinds of things to play nice!) can take a look!

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Give this a try, @zcraber - I've removed as many of the points as I could, but it'll need to be optimised again.
```
m12,1.253c 1.044,0 1.956,0.569 2.441,1.412l9.039,15.623c0.205713,0.337415 0.300563,0.561797 0.325,0.624 0.0995,0.227035 0.191,0.67716 0.191,0.88 0.0025,0.04706 0.004,0.08872 0.004,0.15 -0.0016,1.898656 -1.846649,3.2493 -3.657,2.677l-7.902,-2.607c-0.286708,-0.09475 -0.596292,-0.09475 -0.883,0l-7.9,2.607 -0.163,0.045C1.7215025,23.109778 0.00145065,21.769663 0,19.941 0.0127181,19.438636 0.08671779,19.19614 0.218,18.856 0.2979508,18.655977 0.42079698,18.47709 0.522,18.288 3.535451,13.080787 6.5476826,7.8728687 9.56,2.665 10.061784,1.7913387 10.992493,1.2527481 12,1.253Z
```

**Comment by zcraber** [comment]
Here's what I tried. @PeterShaggyNoble Please check.

```xml
M12,1.253C13.044,1.253 13.956,1.822 14.441,2.665L23.48,18.288C23.686,18.625 24,19.088 24,19.942C24,21.832 22.183,23.182 20.343,22.619L12.441,20.012C12.154,19.917 11.845,19.917 11.558,20.012C11.558,20.012 3.709,22.601 3.658,22.619C2.325,23.099 0.012,22.304 0,19.941C-0.004,19.047 0.421,18.477 0.522,18.288C3.535,13.081 6.548,7.873 9.56,2.665C10.062,1.791 10.992,1.253 12,1.253Z
```

**Comment by PeterShaggyNoble** [comment]
I'm seeing some differences in the 2 bottom corners in that version, @zcraber, most notably in [the left one](https://github.com/simple-icons/simple-icons/assets/15157491/47679f26-e04c-4fb3-94f3-ba2880372d3e). It may simply be an optimisation issue, though, that can be resolved by increasing the precision.

**Comment by PeterShaggyNoble** [comment]
I hope you don't mind, @zcraber, but, in the interests of moving this along, I've pushed an optimised version of my path to the PR.

**Comment by zcraber** [comment]
> I hope you don't mind

Not at all! 😀

File: icons/spaceship.svg
Comment (PratyushJayachandran) [inline icons/spaceship.svg:1]: I still see some differences @PeterShaggyNoble 
spaceship-comparison

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

File: icons/spaceship.svg
Comment (PeterShaggyNoble) [reply icons/spaceship.svg:1]: Looks like you might have inadvertently stretched the source icon in that comparison, @PratyushJayachandran. Will request a review from @service-paradis to double check.

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**